midend/lib/Conversion/MatMulOptimization/CMakeLists.txt

```diff
@@ -11,6 +13,14 @@ add_mlir_library(BatchMatMulOptimization
   BatchMatMulOptimize.cpp
 )
 
+add_mlir_library(BatchMatMulTileOptimization
+  BatchMatMulTileOptimize.cpp
+)
+
+add_mlir_library(BatchMatMulSCFOptimization
+  BatchMatMulSCFOptimize.cpp
+)
+
```

Comment:
Recommend to build `BatchMatMulTileOptimize.cpp` and `BatchMatMulSCFOptimize.cpp` under `BatchMatMulOptimization`, together with existing `BatchMatMulOptimize.cpp`. As suggested before. 
![image](https://github.com/user-attachments/assets/62a49216-cc05-4543-af82-d674e77ceb37)